src/clj/ataru/hakija/hakija_form_service.clj

```diff
@@ -102,6 +102,8 @@
                  (let [cannot-view? (contains? viewing-forbidden-person-info-field-ids
                                                (keyword (:id field)))
                        cannot-edit? (or cannot-view?
+                                        (and (nil? hakukohteet)
```

Comment:
Laittasin tän ehkä mieluummin tonne `editing-allowed-by-hakuaika?` sisään, kun siellä jo tarkastellaan tuota `application-in-processing-state?`:ä.

Reply:
Niin, se vaan laajentaa sitä editing-allowed-by-hakuaika?:n määritelmää ikävästi..